PKHeX/PKM/ShowdownSet.cs

```diff
@@ -144,7 +144,10 @@ public ShowdownSet(string input = null)
             switch (Species)
             {
                 case 658: // Greninja
-                    if (Ability == 210) Form += "-Ash"; // Battle Bond
+                    if (Ability == 210) Form = "Ash"; // Battle Bond
+                    break;
+                case 666: // Vivillon
+                    if (Form == "Pokeball") Form = "Poké Ball";
```

Comment:
This check is there to translate the Pokemon showdown form name to the PKHeX form name.  I think it would be easier to just rename the form in text_Forms_en.txt, since the checks here and on line 284 could be removed.  Due to the nature of the change, however, @kwsch should approve this suggestion first.